.github/workflows/advisories-headers-plugin.yml

```diff
@@ -0,0 +1,81 @@
+name: Advisories headers plugin
+
+on: push
+
+defaults:
+  run:
+    working-directory: wp-content/plugins/advisories-headers
+
+jobs:
+  kahlan:
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        php-versions: ['8.2']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: |
+          echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - uses: actions/cache@v4
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+      - name: Install dependencies
+        run: composer install --no-interaction
+      - run: vendor/bin/kahlan
```

Comment:
Would it be preferable to run the linter, Psalm and the unit tests all in the same job, rather than repeating the setup across three separate jobs?

Reply:
I should probably change that in the template first!